demonstrator/ogc-api-processes/main.js

```diff
@@ -63,15 +119,15 @@ const setProcess = (process) => {
               value: JSON.parse(geoJsonGeom),
               mediaType: "application/geo+json"
             }],
-            cogUrl: "http://nginx/cog/dresden_20220216T1146Z.tif"
+            cogUrl: "http://nginx/cog/dresden/dresden_temperature" + tiffUrl.substring(tiffUrl.lastIndexOf('/'))
```

Comment:
minor: `tiffUrl.substring(tiffUrl.lastIndexOf('/'))` is used 3 times and is not that clear on first sight what happens. Might be a good fit for a util function